corehq/messaging/scheduling/models/abstract.py

```diff
@@ -49,6 +49,7 @@ class Schedule(models.Model):
     domain = models.CharField(max_length=126, db_index=True)
     active = models.BooleanField(default=True)
     deleted = models.BooleanField(default=False)
+    paused = models.BooleanField(default=False)
```

Comment:
Depending on the use case here, assuming we would need to look for paused schedules, do we need index on this? But it wont be often so does not seem worth it. 

There is already an index on domain, so most probably not.
https://stackoverflow.com/a/60635445

Not sure, so just dropping a thought if its worth considering.

Reply:
This will just be used to better handle the pausing of Schedule Items. I don't really think indexing will be worth it